front/tests/008-train-schedule.spec.ts

```diff
@@ -66,9 +66,9 @@ test.describe('Verifying that all elements in the train schedule are loaded corr
     await opTimetablePage.verifyTrainCount(20);
     await opTimetablePage.filterValidityAndVerifyTrainCount(selectedLanguage, 'Invalid', 4);
     await opTimetablePage.filterValidityAndVerifyTrainCount(selectedLanguage, 'All', 20);
-    await opTimetablePage.filterHonoredAndVerifyTrainCount(selectedLanguage, 'Honored', 12);
-    await opTimetablePage.filterValidityAndVerifyTrainCount(selectedLanguage, 'Valid', 12);
-    await opTimetablePage.filterHonoredAndVerifyTrainCount(selectedLanguage, 'Not honored', 4);
+    await opTimetablePage.filterHonoredAndVerifyTrainCount(selectedLanguage, 'Honored', 13);
+    await opTimetablePage.filterValidityAndVerifyTrainCount(selectedLanguage, 'Valid', 13);
+    await opTimetablePage.filterHonoredAndVerifyTrainCount(selectedLanguage, 'Not honored', 3);
```

Comment:
I don't understand those changes in e2e tests (same for margin), maybe we should discuss it aside and just report here the explanation.

Reply:
@Maymanaf could probably explain it better than me, I'm still a little confused

Apparently there was a train that was already really close to the limit, and lowering the speed limit made it possible to follow the margins

Reply:
As far as I understood, there are also probably rounding bugs and changing the base speed-profile may have "solved" the bug for this precise case.
We have some issue tracking those problems and possible evolutions: internal https://github.com/osrd-project/osrd-confidential/issues/680 and also bugs https://github.com/OpenRailAssociation/osrd/issues/9179 + https://github.com/OpenRailAssociation/osrd/issues/8959

I'm resolving it 👍